### PR TITLE
Centers Honkmother on station when summoned by bananium bomb

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

title

#### Changelog

:cl:  
tweak: Honkmother is now centered on station when summoned by bananium bomb
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Checks and Cornflakes

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 1]
```diff
@@ -34,7 +34,21 @@
 
 /obj/machinery/nuclearbomb/syndicate/bananium/really_actually_explode(off_station)
 	//SHE LIVES
-	new /obj/structure/destructible/honkmother(src.loc)
+	var/turf/T = SSmapping.get_station_center()
```
Comment (jcatjmeow) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
	var/turf/center = SSmapping.get_station_center()
```

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 2]
```diff
@@ -34,7 +34,21 @@
...
+	new /obj/structure/destructible/honkmother(T)
+	var/x0 = T.x
```
Comment (jcatjmeow) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
	var/x0 = center.x
```

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 3]
```diff
@@ -34,7 +34,21 @@
...
+	var/y0 = T.y
```
Comment (jcatjmeow) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
	var/y0 = center.y
```

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 4]
```diff
@@ -34,7 +34,21 @@
...
+	for(var/I in spiral_range_turfs(255, T, tick_checked = TRUE))
```
Comment (jcatjmeow) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
	for(var/I in spiral_range_turfs(255, center, tick_checked = TRUE))
```

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 5]
```diff
@@ -34,7 +34,21 @@
...
+		var/turf/T2 = I
+		if(!T2)
+			continue
+		var/dist = cheap_hypotenuse(T2.x, T2.y, x0, y0)
+		if(dist < 100)
+			dist = TRUE
+		else
+			dist = FALSE
+		T.honk_act(dist)
```
Comment (jcatjmeow) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
		center.honk_act(dist)
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]
These suggested changes will stop the duplicate definition.

If you wish, you may alternatively change the name of the var in the for loop below 
```
var/turf/T = get_turf(H)
		if(!T || T.z != z)
```
Or, you could have it just redefine T, in which case the change would be just deleting the "var/turf/" on the above line.

**Comment by ynot01** [comment]
this PR is on hold until honkmother exists

**Comment by ynot01** [comment]
replaced T with center thank you jcat

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 6]
```diff
@@ -34,7 +34,21 @@
 
 /obj/machinery/nuclearbomb/syndicate/bananium/really_actually_explode(off_station)
 	//SHE LIVES
-	new /obj/structure/destructible/honkmother(src.loc)
+	var/turf/center = SSmapping.get_station_center()
+	new /obj/structure/destructible/honkmother(T)
```
Comment (ynot01) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]:
```suggestion
	new /obj/structure/destructible/honkmother(center)
```

**Review by fluffe9911 — APPROVED** [review]
ok

**Pull request merged**